wled00/FX.cpp

```diff
@@ -1375,15 +1419,17 @@ uint16_t mode_loading(void) {
 static const char _data_FX_MODE_LOADING[] PROGMEM = "Loading@!,Fade;!,!;!;;ix=16";
 
 
-//American Police Light with all LEDs Red and Blue
-uint16_t police_base(uint32_t color1, uint32_t color2) {
-  if (SEGLEN == 1) return mode_static();
+//Police Lights with custom colors
+uint16_t mode_two_dots() {
+ if (SEGLEN == 1) return mode_static();
   unsigned delay = 1 + (FRAMETIME<<3) / SEGLEN;  // longer segments should change faster
   uint32_t it = strip.now / map(SEGMENT.speed, 0, 255, delay<<4, delay);
   unsigned offset = it % SEGLEN;
-
   unsigned width = ((SEGLEN*(SEGMENT.intensity+1))>>9); //max width is half the strip
   if (!width) width = 1;
+  if (!SEGMENT.check2) SEGMENT.fill(SEGCOLOR(2));
+  uint32_t color1 = SEGCOLOR(0);
```

Comment:
for me it's less about caring and more a lack of knowledge when and how to use it properly (plus in short, contained functions: does it matter apart from readability?)

Reply:
`const` is easy: just stick it to every variable, then remove it again if it doesn't compile, and the reason for that is that you actually mean to modify it multiple times :)

Does it matter? The answer to that is really the same for most programming good practices:

* It *probably* doesn't matter for performacne.
* It may or may not increase readability, depending on who you ask:
  * It does state intent, and that does simplify a readers process to understand the code.
  * It gives the reader some more information that he doesn't have to look up, reducing the mental load.
  * It clutters the code, and too much noise can hinder fast reading.
* It however protects you and future maintainers from unintentional actions. In the case of `const` from modifying a variable that is supposed to be constant. You can obviously remove the `const` later if necessary, but you force that decision to be made consciously.

The length of the context is a consideration that I have only ever heard about variable naming before. I think the thing about good practices is that you have to make them a habit, and therefore the scope doesn't matter. If you do it everywhere, it also improves consistency, which is always good.